### PR TITLE
DISTX-218 - Added sql files for user sync for async operation

### PR DESCRIPTION
**Pull request by abhishekaypurohit**
Added schema related sql files for user sync for async operation

Related to: https://jira.cloudera.com/browse/DISTX-218 and https://jira.cloudera.com/browse/DISTX-111

`
BUILD SUCCESSFUL in 12m 41s
448 actionable tasks: 442 executed, 6 up-to-date`

@handavid @keyki @lacikaaa

### CODE REVIEW

File: freeipa/src/main/resources/schema/app/20190625150000_create_user_sync_env_details.sql [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        CONSTRAINT CONSTRAINT usersyncenvdetails_pkey
+            PRIMARY KEY,
+    usersyncsummaryid  bigint not null
+      constraint fk_usersyncsummary_id
+        references usersyncsummary,
+    envcrn VARCHAR(255),
+    umspulltime int8 NOT NULL DEFAULT '-1'::integer,
+    endtime int8 NOT NULL DEFAULT '-1'::integer,
+    status VARCHAR(255),
+    detail_status_obj  VARCHAR(255)
```
Comment (handavid) [inline freeipa/src/main/resources/schema/app/20190625150000_create_user_sync_env_details.sql]: I'm guessing that we'll need more than 255 char for this

File: freeipa/src/main/resources/schema/app/20190624180000_create_user_sync_summary.sql [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+-- //@UNDO
+
+DROP TABLE IF EXISTS usersyncsummary;
+
+DROP SEQUENCE IF EXISTS usersyncsummary_id_seq;
+--------------------------------
+
+
+CREATE INDEX uss_id_accountid_idx
```
Comment (handavid) [inline freeipa/src/main/resources/schema/app/20190624180000_create_user_sync_summary.sql]: this should go above the UNDO line

File: freeipa/src/main/resources/schema/app/20190625150000_create_user_sync_env_details.sql [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+-- //@UNDO
+
+DROP TABLE IF EXISTS usersyncenvdetails;
+
+DROP SEQUENCE IF EXISTS usersyncenvdetails_id_seq;
+--------------------------------
+
+
+CREATE INDEX usersed_id_usersyncsummaryid_idx
```
Comment (handavid) [inline freeipa/src/main/resources/schema/app/20190625150000_create_user_sync_env_details.sql]: this should go above the UNDO line

File: freeipa/src/main/resources/schema/app/20190624180000_create_user_sync_summary.sql [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+CREATE SEQUENCE usersyncsummary_id_seq START WITH 1
+  INCREMENT BY 1
+  NO MINVALUE
+  NO MAXVALUE
+  CACHE 1;
+
+
+
+CREATE TABLE IF NOT EXISTS usersyncsummary
```
Comment (handavid) [inline freeipa/src/main/resources/schema/app/20190624180000_create_user_sync_summary.sql]: Can you add the corresponding Entity classes in the same commit?

**Comment by lacikaaa** [comment]
I don't see the purpose of the SQL scripts without the java code, please open PR's where the code and the necessary SQL scripts both are in place so we can validate them

**Pull request closed without merging**